include/picongpu/particles/fusion/fieldSlots.hpp

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2022-2024 Pawel Ordyna
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "picongpu/defines.hpp"
+
+namespace picongpu
+{
+    namespace particles
+    {
+        namespace fusion
+        {
+            //! FieldTmp slot which is used to store the calculated Debye length
+            constexpr uint32_t screeningLengthSlot = 0u;
```

Comment:
If I not missed something this variable is not used, therefore the file can be removed.